src/test/java/com/wikia/webdriver/testcases/auth/LoginTests.java

```diff
@@ -1,74 +1,253 @@
 package com.wikia.webdriver.testcases.auth;
 
+import com.wikia.webdriver.common.contentpatterns.MercurySubpages;
+import com.wikia.webdriver.common.contentpatterns.MercuryWikis;
+import com.wikia.webdriver.common.core.Assertion;
 import com.wikia.webdriver.common.core.Helios;
 import com.wikia.webdriver.common.core.annotations.Execute;
-import com.wikia.webdriver.common.core.api.ArticleContent;
-import com.wikia.webdriver.common.core.configuration.Configuration;
+import com.wikia.webdriver.common.core.annotations.InBrowser;
+import com.wikia.webdriver.common.core.drivers.Browser;
+import com.wikia.webdriver.common.core.helpers.Emulator;
 import com.wikia.webdriver.common.core.helpers.User;
-import com.wikia.webdriver.common.properties.Credentials;
 import com.wikia.webdriver.common.templates.NewTestTemplate;
+import com.wikia.webdriver.elements.mercury.components.TopBar;
+import com.wikia.webdriver.elements.mercury.pages.ArticlePage;
+import com.wikia.webdriver.elements.mercury.pages.discussions.PostsListPage;
 import com.wikia.webdriver.pageobjectsfactory.componentobject.global_navitagtion.NavigationBar;
-import com.wikia.webdriver.pageobjectsfactory.pageobject.WikiBasePageObject;
 import com.wikia.webdriver.pageobjectsfactory.pageobject.article.ArticlePageObject;
 import com.wikia.webdriver.pageobjectsfactory.pageobject.auth.signin.DetachedSignInPage;
-
+import com.wikia.webdriver.pageobjectsfactory.pageobject.auth.signin.SignInPage;
 import org.testng.annotations.Test;
 
+import java.time.Instant;
+
 import static com.wikia.webdriver.common.core.Assertion.assertTrue;
+import static org.testng.Assert.assertFalse;
 
-@Test(groups = "auth-login")
+@Execute(onWikia = MercuryWikis.MERCURY_AUTOMATION_TESTING)
 public class LoginTests extends NewTestTemplate {
 
-  Credentials credentials = Configuration.getCredentials();
+  private User user = User.LOGIN_USER;
+  private User japaneseUser = User.USER_JAPAN;
+  private User staff = User.LOGIN_STAFF;
+
+  private static final String DESKTOP = "auth-login-desktop";
+  private static final String MOBILE = "auth-login-mobile";
+
+  private static final String ERROR_MESSAGE =
+    "We don't recognize these credentials. Try again or register a new account.";
+  private static final String SUBMIT_BUTTON_DISABLED_MSG = "Submit button should be disabled";
+
+
+  @Test(groups = MOBILE)
+  @InBrowser(browser = Browser.CHROME, emulator = Emulator.GOOGLE_NEXUS_5)
+  public void userCanCloseJoinPage() {
+    ArticlePage article = openArticleOnMobile();
+    String title = article.getArticleTitle();
+    article
+      .getTopBar()
+      .openNavigation()
+      .clickOnSignInRegisterButton()
+      .close();
+    Assertion.assertEquals(article.getArticleTitle(), title);
```

Comment:
`assertThat(article.getArticleTitle()).isEqualTo(previousTitle)`?

Reply:
we don't have `AssertJ` yet :cry: